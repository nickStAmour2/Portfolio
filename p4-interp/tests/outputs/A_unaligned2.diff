--- outputs/A_unaligned2.txt	2022-12-13 15:08:03.228420657 -0500
+++ expected/A_unaligned2.txt	2022-09-29 09:42:12.000000000 -0400
@@ -1,8 +1,8 @@
-Beginning execution at 0x0100
+Beginning execution at 0x0555
 Y86 CPU state:
   %rip: 0000000000000574   flags: Z0 S0 O0     HLT
-  %rax: 0000000000000000    %rcx: 0000000000000000
-  %rdx: 0000000000000000    %rbx: 0000000000000000
+  %rax: 000000000000014d    %rcx: 000000000000006f
+  %rdx: 0000000000000000    %rbx: 00000000000000de
   %rsp: 0000000000000000    %rbp: 0000000000000000
   %rsi: 0000000000000000    %rdi: 0000000000000000
    %r8: 0000000000000000     %r9: 0000000000000000
