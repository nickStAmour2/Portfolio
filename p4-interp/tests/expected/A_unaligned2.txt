Beginning execution at 0x0555
Y86 CPU state:
  %rip: 0000000000000574   flags: Z0 S0 O0     HLT
  %rax: 000000000000014d    %rcx: 000000000000006f
  %rdx: 0000000000000000    %rbx: 00000000000000de
  %rsp: 0000000000000000    %rbp: 0000000000000000
  %rsi: 0000000000000000    %rdi: 0000000000000000
   %r8: 0000000000000000     %r9: 0000000000000000
  %r10: 0000000000000000    %r11: 0000000000000000
  %r12: 0000000000000000    %r13: 0000000000000000
  %r14: 0000000000000000
Total execution count: 4
